packages/sui-theme/src/components/thumbnail/_basic.scss

```diff
@@ -0,0 +1,2 @@
+$jc-thumbnail-basic-image: flex-start !default;
+$ar-thumbnail-basic-image: auto !default;
```

Comment:
what does `ar-` mean?

Reply:
aspect-ratio

Reply:
`aspect-ratio` I didn't find the abbreviation 😓